opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -190,6 +190,15 @@ def _transcribe(item, parent=None, editRate=24, masterMobs=None):
     metadata["Name"] = _get_name(item)
     metadata["ClassName"] = _get_class_name(item)
 
+    # Some AAF objects (like TimelineMobSlot) have an edit rate
+    # which should be used for all of the object's children.
+    # We will pass it on to any recursive calls to _transcribe()
+    if hasattr(item, "edit_rate"):
+        editRate = float(item.edit_rate)
+        # Only use a float if really needed.
```

Comment:
Could there be a comment here on why this is needed? (pyaaf2 borks for some reason on floats). I think I've seen this borking before. Maybe we should talk to Mark about it.

Reply:
I'll file a separate issue to track that down.

Reply:
Actually, I think I found the problem in the writer and fixed it.